.asf.yaml

```diff
@@ -0,0 +1,41 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+github:
+    description: ingress controller for K8s
+    homepage: https://apisix.apache.org/
+    labels:
+      - ingress
+      - controller
+      - Kubernetes
+      - K8s
+      - apigateway
+      - microservices
+      - api
+      - loadbalancing
+      - apisix
+      - devops
+
+    enabled_merge_buttons:
+      squash:  true
+      merge:   false
+      rebase:  false
+
+    notifications:
```

Comment:
`notifications` should be in the first level of yaml

Reply:
why? 

`APISIX` AND `APISIX Dashboard`, they are using the same way.

https://github.com/apache/apisix/blob/master/.asf.yaml#L45-L48
https://github.com/apache/apisix-dashboard/blob/master/.asf.yaml#L33-L36

Reply:
![image](https://user-images.githubusercontent.com/6814606/101245271-98756180-3746-11eb-8b45-0a5720752e87.png)

wow @moonming do you know this?